### ISSUE TITLE
CockroachDB cluster status Suspect

### ISSUE THREAD

**Issue by EamonZhang** [cockroachdb/cockroach#17775]
version v1.0.4;
3 nodes cluster ; 10.1.88.42 crdb02 , 10.1.88.43 crdb03, 10.1.88.44 crdb04

  https://ip:8080/#/cluster/all/overview

```
Summary
Total Nodes View nodes list 3
Healthy 0
Suspect 3
Dead 0

```

How to ask for help when  in trouble?

[cockroach.crdb02.root.2017-08-21T15_11_12+08_00.027316.log.txt](https://github.com/cockroachdb/cockroach/files/1238085/cockroach.crdb02.root.2017-08-21T15_11_12.08_00.027316.log.txt)
[cockroach.crdb03.root.2017-08-21T15_15_53+08_00.006986.log.txt](https://github.com/cockroachdb/cockroach/files/1238086/cockroach.crdb03.root.2017-08-21T15_15_53.08_00.006986.log.txt)
[cockroach.crdb04.root.2017-08-21T15_21_18+08_00.006201.log.txt](https://github.com/cockroachdb/cockroach/files/1238087/cockroach.crdb04.root.2017-08-21T15_21_18.08_00.006201.log.txt)

**Comment by EamonZhang** [comment]
start only one node in insecure mode.

```
cockroach version
Build Tag:    v1.0.4
Build Time:   2017/07/27 17:54:36
Distribution: CCL
Platform:     linux amd64
Go Version:   go1.8.3
C Compiler:   gcc 6.3.0
Build SHA-1:  5b757262d33d814bda1deb2af20161a1f7749df3
Build Type:   release
```

```
uname -a
Linux crdb02 3.10.0-514.26.2.el7.x86_64 #1 SMP Tue Jul 4 15:04:05 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```


[cockroach.crdb02.root.2017-08-22T16_54_31+08_00.002434.log.txt](https://github.com/cockroachdb/cockroach/files/1241424/cockroach.crdb02.root.2017-08-22T16_54_31.08_00.002434.log.txt)


cockroach start --background --insecure --host=10.1.88.42

**Comment by dianasaur323** [comment]
@EamonZhang here is a tracking issue: https://github.com/cockroachdb/cockroach/issues/17840

**Comment by a-robinson** [comment]
Closed by #18260
